packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -522,6 +522,7 @@ const SummaryCard = decorate([
 
         return (totalSize * state.numberOfHostsWithDisks) / state.replication.value
       },
+      replicationCount: state => state.replication.value > state.numberOfHostsWithDisks,
```

Comment:
The name of the computed is a bit confusing. It looks like it returns a `number`.
What do you think of something like: 
- `replicationCountHigherThanHostsWithDisks`
- `isReplicationValid`